pkg/shell/base_index.js

```diff
@@ -140,14 +147,31 @@ var phantom_checkpoint = phantom_checkpoint || function () { };
                 frame.style.display = "none";
 
                 var base, checksum;
-                if (machine)
-                    checksum = machine.checksum;
-                if (host === "localhost")
-                    base = "..";
-                else if (checksum)
-                    base = "../../" + checksum;
-                else
+                if (machine) {
+                    if (machine.manifests && machine.manifests[".checksum"])
+                        checksum = "$" + machine.manifests[".checksum"];
```

Comment:
This is not intended to change, and the tests are checking that it stays the same.